### PR TITLE
feat(jest): 100% coverages

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

100% coverage for store/accounts/mutations.test.ts
100% coverage for store/sounds/actions.test.ts

**Which issue(s) this PR fixes** 🔨

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: store/accounts/mutations.test.ts [hunk 1]
```diff
@@ -40,7 +42,7 @@ describe('init', () => {
   })
 
   it('should setPin', () => {
-    const localStateForUnitTest = { ...state }
+    const localStateForUnitTest = InitialAccountsState()
```
Comment (WanderingHogan) [inline store/accounts/mutations.test.ts:45]: would this be a good change to use beforeEach?

```
  let localStateForUnitTest

  beforeEach(() => {
    localStateForUnitTest = InitialAccountsState()
  })
```

File: store/sounds/actions.test.ts [hunk 2]
```diff
@@ -89,6 +89,27 @@ describe('Manage sounds', () => {
     )
   })
 
+  test('sound stops but argument is empty array', () => {
+    const result: any = inst.stopSounds(
+      {
+        state: {
+          newMessage: true,
```
Comment (WanderingHogan) [inline store/sounds/actions.test.ts:96]: you could clean this up a lot if you made an object at the top somewhere like this:

```
       const state = {
          newMessage: true,
          hangup: true,
          call: true,
          mute: true,
          unmute: true,
          deafen: true,
          undeafen: true,
          upload: true,
          connected: true,
        },
```

and then in the object you pass in, if you want undeafen to be false, for example

```
{
  state: {...state, undeafen: false}
}
        ```

**Review by WanderingHogan — CHANGES_REQUESTED** [review]

File: store/sounds/actions.test.ts (hunk 2 above)
Comment (drepram) [reply store/sounds/actions.test.ts]: resolved!

File: store/accounts/mutations.test.ts (hunk 1 above)
Comment (drepram) [reply store/accounts/mutations.test.ts]: resolved!

**Comment by stavares843** [comment]
🎉 

feel free to update the label for missing review for hogan to re check 🔨 and thanks for updating 🎉

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**